backends/s3/s3testing/port.go

```diff
@@ -0,0 +1,14 @@
+package s3testing
+
+import "net"
+
+// FreePort returns a port number free for use.
```

Comment:
Nit: slight race, as after closing it, you can no longer assume it is still free. But I do not have any better solution. Passing port 0 and then figuring out what port minio is actually listening on would likely be more error prone.